.github/workflows/java-ci.yml

```diff
@@ -65,8 +65,8 @@ jobs:
     timeout-minutes: 60
     needs: build
     runs-on: ubuntu-latest
-    # Don't run for forks because of missing secrets
-    if: (success() || failure()) && !(github.event.pull_request && github.event.pull_request.head.repo.fork)
+    # Don't run for forks and dependabot because of missing secrets
+    if: (success() || failure()) && !(github.event.pull_request && github.event.pull_request.head.repo.fork) && github.actor != 'dependabot[bot]'
```

Comment:
Do you know why we need the first condition `success() || failure()`? Sounds like it can be removed completely or do we have a third option?

Reply:
No, it was there from the beginning. The [action itself](https://github.com/EnricoMi/publish-unit-test-result-action) has `always()` in the example but without additional conditions. I don't know how GH actions evaluate the condition when there is no reference to the previous state in the if clause.